experimentation/image_compression.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import pathlib
+import cv2
+import imagej
+import skimage
+import sys
+import os 
+import matplotlib.pyplot as plt
+import numpy as np
+from IPython.utils.io import capture_output
+import warnings
+from PIL import Image
+sys.path.append(os.path.abspath('./'))
+from processor import BasicpyPreprocessor
+import PyBaSiC.pybasic
+
+fiji_path = pathlib.Path("/home/keenanmanpearl/Desktop/mitocheck/fiji-linux64/Fiji.app")
+movie_path = pathlib.Path(
+    "/home/keenanmanpearl/Desktop/mitocheck_movies/movies/LT0001_02/1.ch5/00001_01.ch5"
+)
```

Comment:
```suggestion
base_dir = pathlib.Path("/home/keenanmanpearl/Desktop")

fiji_path = pathlib.Path(f"{base_dir}/mitocheck/fiji-linux64/Fiji.app")
movie_path = pathlib.Path(
   f"{base_dir}/mitocheck_movies/movies/LT0001_02/1.ch5/00001_01.ch5"
)
```

Please keep in mind [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Please also note that I am making a "suggestion" (the technical term on github) on the `.py` file. Please port all changes to the `.ipynb` file and then run the `nbconvert`